downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -113,9 +113,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### `jwt` package
 
-- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+- Added support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` values in JWT signature configurations. With this update, the `config` field of `jwt:IssuerSignatureConfig` now allows `crypto:PrivateKey`, and the `certFile` field of `jwt:ValidatorSignatureConfig` now allows `crypto:PublicKey`. With these additions, union types (or whatever that is not supported now) will be disallowed.
```

Comment:
or whatever that is not supported now -> I meant to rephrase this to specify what is not allowed now :) Not include it in the release note.

Reply:
As we have already mentioned the previous and new types in the code-samples, IMO we can mention that previously specified union types are not allowed in the text. Updated the PR with following commit [1] 

[1] - [406b2bc3d4bcd3961a7856a1849d5eeab3441d61](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/406b2bc3d4bcd3961a7856a1849d5eeab3441d61)